File: docs/legacy(old PACE)/C#/3.5/PaceAPISample/InvokeProcessTest.cs
using System;

using efipaceservices;

namespace Pace_Web_Service_SDK
{
	public class InvokeProcessTest
	{
		public static void Run()
		{
            InvokeProcessHttpBinding invokeProcessHttpBinding = PaceClient.getInvokeProcessHttpBinding();

            /*ProcessResults results = invokeProcessHttpBinding.postInventoryTrn("true");
            Console.WriteLine("Posted " + results.successes.Length + " inventory txn");*/

            ProcessResults results = invokeProcessHttpBinding.postGLBatchTrn("@approved");
			Console.WriteLine("GL Batch Trn Post was a " + ( (bool)results.successful ? "success" : "failure"));

			if (results.successes.Length != 0)
			{

                foreach( SuccessProcessItem item in results.successes )
                {
                    Console.WriteLine("Sucesses: " + item.reason);				    
                }
			}
			
			if (results.failures.Length != 0)
			{
				Console.WriteLine("Failures: ");
				foreach (FailedProcessItem failure in results.failures)
					Console.WriteLine(failure.reason);
			}
		}
	}
}

File: docs/legacy(old PACE)/unsupported/VB.NET/PaceWebServiceSDK/sample/InvokeProcessTest.vb
Imports PaceWebServiceSDK.efipaceservices

Public Class InvokeProcessTest
    Public Shared Sub Run()

        Dim invokeProcessHttpBinding As InvokeProcessHttpBinding = PaceClient.getInvokeProcessHttpBinding()

        Dim results As ProcessResults = invokeProcessHttpBinding.postGLBatchTrn("@approved")
        Console.WriteLine("GL Batch Post was a " & (If(CBool(results.successful), "success", "failure")))

        If results.successes.Length <> 0 Then
            For Each item As SuccessProcessItem In results.successes
                Console.WriteLine("Sucesses: " + item.reason)
            Next
        End If

        If results.failures.Length <> 0 Then
            For Each item As FailedProcessItem In results.failures
                Console.WriteLine("Failures: " + item.reason)
            Next
        End If

    End Sub
End Class
